src/App.tsx

```diff
@@ -1,25 +1,39 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
+import { Pagination } from './components/Pagination';
 import { getNumbers } from './utils';
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [perPage, setPerPage] = useState(5);
+  const [currentPage, setCurrentPage] = useState(1);
+  const totalItems = items.length;
+
+  const from = (currentPage - 1) * Number(perPage) + 1;
```

Comment:
Casting `perPage` to a number here is redundant since its state is already controlled and should always be a number. Ensure that the state type is set correctly to avoid unnecessary type casting.